Content.Server.Database/Model.cs

```diff
@@ -1119,4 +1123,16 @@ public class RoleWhitelist
         [Required]
         public string RoleId { get; set; } = default!;
     }
+
+    [Table("text_automod")]
+    public sealed class TextAutomodFilter
```

Comment:
Why is this class named differently from the DB table?

Reply:
Renamed to TextAutomod in [f399796](https://github.com/space-wizards/space-station-14/pull/28570/commits/f39979656b52d71282f883a90820a5a0fb1c2d3f)